slf4j-jcl/pom.xml

```diff
@@ -21,33 +22,11 @@
       <groupId>org.slf4j</groupId>
       <artifactId>slf4j-api</artifactId>
     </dependency>
-
     <dependency>
       <groupId>commons-logging</groupId>
       <artifactId>commons-logging</artifactId>
-      <version>1.1.1</version>
+      <version>1.2</version>
```

Comment:
Not sure it compiled ok and test cases workef.  This one is for writing to 1.2 rather than jcl to slf4j.  I have used commons logging 1.2 without issues elsewhere in projects using it but not slf4j without issues.  I assume since unit tests all pass it is good.  I can remove that piece if you want me to.

--- Original Message ---

From: "Matt Bishop" <notifications@github.com>
Sent: March 16, 2015 10:40 AM
To: "qos-ch/slf4j" <slf4j@noreply.github.com>
Cc: "Jeremy Landis" <jeremylandis@hotmail.com>
Subject: Re: [slf4j] Updating builds (#88)

>      <dependency>
>        <groupId>commons-logging</groupId>
>        <artifactId>commons-logging</artifactId>
> -      <version>1.1.1</version>
> +      <version>1.2</version>

Not sure about this update. It appears more more than the version # needs to be performed. See #89 for discussion.

---
Reply to this email directly or view it on GitHub:
https://github.com/qos-ch/slf4j/pull/88/files#r26489554